torch/onnx/_type_utils.py

```diff
@@ -170,7 +170,9 @@ def from_value(
             SymbolicValueError: when value.type()'s info are empty and default is None
         """
 
-        if not isinstance(value, (torch._C.Value, torch.Tensor)):
+        if not isinstance(value, (torch._C.Value, torch.Tensor)) or (
+            isinstance(value, torch._C.Value) and value.node().mustBeNone()
+        ):
```

Comment:
Was this logic used in this change? Maybe in _try_get_scalar_type?

Reply:
It was not used, but still needed for other scenarios if TorchScript behavior is not fixed